<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_one</class>
 <widget class="QWidget" name="Set_one">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {background-color: transparent;
border-radius: 10px;
}
Button_1{
background-color:black;
}</string>
  </property>
  <widget class="QPushButton" name="Button_main">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>640</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>34</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 170, 0);
border:4px solid black;
border-radius: 10px;
}
QPushButton:hover{
background-color: rgb(92, 176, 255);
border:4px solid black;
border-radius: 10px;
}
QPushButton:pressed{
background-color: rgb(85, 255, 127);
border:4px solid black;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_ok">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>640</y>
     <width>180</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>34</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 170, 0);
border:4px solid black;
border-radius: 10px;
}
QPushButton:hover{
background-color: rgb(92, 176, 255);
border:4px solid black;
border-radius: 10px;
}
QPushButton:pressed{
background-color: rgb(85, 255, 127);
border:4px solid black;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_me">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255,255,255);
border-radius: 75px;
}
QPushButton:hover{
background-color: red;
border-radius: 75px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Image_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>352</width>
     <height>385</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>352</width>
     <height>385</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255,255,255);
border-radius: 45px;
}
QPushButton:hover{
background-color: rgb(255, 170, 127);
border-radius: 45px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Image_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>100</y>
     <width>385</width>
     <height>385</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>385</width>
     <height>385</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255,255,255);
border-radius: 45px;
}
QPushButton:hover{
background-color: rgb(255, 170, 255);
border-radius: 45px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>120</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>47</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>当前小鸟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>561</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小鸟选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/resorce/icon.ico)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>761</width>
     <height>122</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Button_1">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-image: url(:/resorce/kaituo.png);
}
QPushButton:hover {
	border-image: url(:/resorce/kaituojin.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_2">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-image: url(:/resorce/liuying.png);
}
QPushButton:hover {
	border-image: url(:/resorce/liuyingjin.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_3">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-image: url(:/resorce/sanyue.png);
}
QPushButton:hover {
	border-image: url(:/resorce/sanyuejin.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_4">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-image: url(:/resorce/yinlang.png);
}
QPushButton:hover {
	border-image: url(:/resorce/yinlangjin.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_5">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-image: url(:/resorce/zhigeng.png);
}
QPushButton:hover {
	border-image: url(:/resorce/zhigengjin.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Button_left">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>81</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>141</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/resorce/jiantouzuojin.png);
}
QPushButton:hover {
	border-image: url(:/resorce/jiantouzuo.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_right">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>230</y>
     <width>81</width>
     <height>141</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>141</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/resorce/jiantouyoujin.png);
}
QPushButton:hover {
	border-image: url(:/resorce/jiantouyou.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Button_left</zorder>
  <zorder>Button_right</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>Button_main</zorder>
  <zorder>Button_ok</zorder>
  <zorder>Button_me</zorder>
  <zorder>Image_1</zorder>
  <zorder>Image_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
